src/test/java/performance/DFSelectionSetPerformance.java

```diff
@@ -1,27 +1,15 @@
-package benchmark;
+package performance;
 
+import benchmark.BenchmarkUtils;
 import graphql.execution.CoercedVariables;
 import graphql.language.Document;
 import graphql.normalized.ExecutableNormalizedField;
 import graphql.normalized.ExecutableNormalizedOperation;
 import graphql.normalized.ExecutableNormalizedOperationFactory;
 import graphql.parser.Parser;
-import graphql.schema.DataFetchingFieldSelectionSet;
-import graphql.schema.DataFetchingFieldSelectionSetImpl;
-import graphql.schema.GraphQLOutputType;
-import graphql.schema.GraphQLSchema;
-import graphql.schema.SelectedField;
+import graphql.schema.*;
```

Comment:
A nit: could you please change it back to individual imports rather than the wildcard? Thanks

Reply:
Fixed!